test/e2e/index.test.ts

```diff
@@ -37,6 +64,32 @@ describe('e2e', () => {
   const baseUrl = 'http://localhost:5000';
   let browser: playwright.Browser;
   let page: playwright.Page;
+  const screen: PlaywrightScreen = {
+    getByLabelText: (...inputArgs) => {
+      return page.evaluateHandle(
+        (args) => window.DomTestingLibrary.getByLabelText(document.body, ...args),
+        inputArgs,
+      );
```

Comment:
Imagine that code without rest parameters :smile:

```suggestion
    getByLabelText: () => {
      return page.evaluateHandle(
        (args) => window.DomTestingLibrary.getByLabelText.apply(undefined, [document.body].concat(args)),
        Array.prototype.slice.call(arguments),
      );
```